### PR TITLE
[SLDEV-7125] Sealights CLI - Pull request issues

### PR DESCRIPTION
**Pull request by RoniSegal**

### CODE REVIEW

File: src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java [hunk 1]
```diff
@@ -49,9 +50,8 @@ public void execute_hasTechOptions_shouldAdd() throws IOException {
         //Arrange
         String packagesIncluded = "io.include.*";
         String packagesExcluded = "io.exclude.*";
-        DescribableList<TechnologyOptions, TechnologyOptionsDescriptor> technologyOptions = new DescribableList<>(JavaOptions.DescriptorImpl.NOOP);
-        technologyOptions.add(new JavaOptions(packagesIncluded, packagesExcluded));
-        PrConfigCommandExecutor executor = createTestedExecutor(technologyOptions);
+        TECH_OPTIONS.add(new JavaOptions(packagesIncluded, packagesExcluded));
```
Comment (nadavye) [inline src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java:53]: Where do you reset the `TECH_OPTIONS`? I have a feeling it is shared between the tests.

File: src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigCommandExecutor.java [hunk 2]
```diff
@@ -31,6 +31,7 @@ protected void addBaseArgumentsLine(List<String> commandsList) {
             addArgumentKeyVal("tokenfile", baseArgs.getTokenFile(), commandsList);
         }
         addArgumentKeyVal("appname", baseArgs.getAppName(), commandsList);
+        addArgumentKeyVal("proxy", baseArgs.getProxy(), commandsList);
```
Comment (nadavye) [inline src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigCommandExecutor.java:34]: Looks good.
Please see that:
1. We are not missing any other fields.
2. Proxy is working on other commands as well.
3. We are not missing any other fields the rest of the commands.

Comment (RoniSegal) [reply src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigCommandExecutor.java:34]: 1. A PR-BSID is created - it means it works well.
2. Tested - seems that it works well
3. I cant tell, you want me to do a full QA cycle? this plugin is running for while and we did not hear a lot of missing fields

File: src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java (hunk 1 above)
Comment (RoniSegal) [reply src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java]: Done

**Pull request merged**